conjure_python_client/_http/requests_client.py

```diff
@@ -73,6 +79,8 @@ def _request(self, *args, **kwargs):
                         if isinstance(value, bool) else str(value)
                 kwargs[param_kind] = cleaned_params
 
+        kwargs['timeout'] = (self._connect_timeout, self._read_timeout)
```

Comment:
why is this a tuple? The [documentation](http://docs.python-requests.org/en/master/user/quickstart/#timeouts) seems to suggest it should just be a number

Reply:
tuple allows specifying connect and read timeouts separately http://docs.python-requests.org/en/master/user/advanced/#timeouts